guides/get-going-with-gitops.mdx

```diff
@@ -0,0 +1,485 @@
+If no rules match, then the flags top-level `enabled` property is used as the final default return value.
+
+```yaml features.yml
+flags:
+- key: use-quicksort-algorithm
+# ...
+  - segment:
+      key: internal-users
+      value: true
+  - threshold:
+      percentage: 20
+      value: true
+```
+
+This threshold percentage is set to `20`, meaning roughly `20%` of entity IDs will match and cause the flag to return `enabled = true`.
+
+<Info>
+  Head to [Concepts: Bucketing](/concepts#bucketing) to learn how this mechanism
+  is actually achieved.
+</Info>
+
+#### Proposing and Integrating Our Change
+
+Once again, we add, commit and push the change to our branch.
+
+```bash
+git add features.yml
+
+git commit -m "feat: enable use-quicksort-algorithm for 20% of users"
+
+git push enable-flag-for-20-percent
+```
+
+From here, we can open a pull-request and get feedback from our team.
+Once we've have an approval, we can merge the PR and the change goes live.
+
+### Closing the Loop
+
+This process can be repeated, each time increasing the `percentage` property of this newly added `threshold` rollout rule.
+Once we get to the stage of enabling the flag for 100% of users, we can either set the percentage to `100` or we can remove the targeting rules altogether and set the `enabled` property to `true`.
+
+```yaml features.yml
+version: "1.2"
+namespace: default
+flags:
+  - key: use-quicksort-algorithm
+    type: BOOLEAN_FLAG_TYPE
+    enabled: true
+```
+
+Beyond this, we can further close the loop by removing the feature flag call from our application code and simply use the new `quicksort` function.
+
+```go pkg/server/words.go
+func (s *Server) ListWords(w http.ResponseWriter, r *http.Request) {
+  words, err := getWords(r.Context())
+  if err != nil {
+    http.Error(w, err.Error(), http.StatusInternalServerError)
+    return
+  }
+
+  quicksort(words)
+
+  if err := json.NewEncoder(w).Encode(words); err != nil {
+    http.Error(w, err.Error(), http.StatusInternalServerError)
+    return
+  }
+}
+```
+
+## Recap
+
+We've successfully rolled a feature out to production using GitOps practices via Flipt's declarative feature flag configuration files and filesystem backends.
+Along the way, we had the opportunity to use Git to understand the current state of the world.
+At each commit, we could've fully recreate the configuration of our entire application, including the state of Flipt itself.
+
+### Further Considerations
+
+Now you have all the tools necessary to practice GitOps with your feature flags.
+
+However, in production, you mileage may vary with the different backed types.
+You might want to consider the `object` (and AWS S3 type) backend if your source Git repositories are large (we're working on a guide for that now).
+
+The filesystem storage backends currently mandate that the UI is **read-only**.
+We've thoughts on how this could change in the future, but for now this is a limitation.
```

Comment:
```suggestion
We have thoughts on how this could change in the future, but for now, this is a limitation.
```

Reply:
Vales not gonna like this

Reply:
lol i think its ok in this case? `we've have` doesn't make sense, maybe `we've got`

Reply:
Yeah sorry the we've have was just me borking a huge match / replace.
I've gone with we've, just not we have.